src/component/chatting/chattingContent.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import Talk from './talk';
+import { ChattingRoom } from './../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { chatList } from './../../atom/atom';
+import { useRef, useEffect } from 'react';
+
+const ChattingContent = ({ chattingRoomId }: ChattingRoom) => {
+  const chattingList = useRecoilValue(chatList);
+
+  const scrollRef = useRef<HTMLDivElement | null>(null);
+
+  const scrollBottom = () => {
+    if (scrollRef.current) {
+      scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
+    }
+  };
+
+  useEffect(() => {
+    scrollBottom();
+  }, [chattingList[chattingRoomId]]);
```

Comment:
```suggestion
  }, [chattingList]);
```
<img width="747" alt="image" src="https://user-images.githubusercontent.com/65931227/236628835-26ed2373-f30c-45a2-a900-7e6d67ab6a13.png">

그렇다고 합니다... 

요 [링크](https://github.com/facebook/react/issues/14920) 를 보면 불필요하게 해당 인덱스에 위치한 배열까지 타고 들어가서 변화를 감지하는 복잡한 로직을 지양하는 것 같아요!

Reply:
헉 이렇게 디테일한 코멘트 감사합니다!!! 여기까진 차마 신경쓰지 못 했네요ㅠㅠ